tests/routers/test_p2p.py

```
@@ -0,0 +1,137 @@
+"""P2P API tests."""
+
+import json
+
+from app.models import P2PPeer, P2PPeerStatus, RemoteSharedFileCache, StorageType
+
+
+def _parse_ndjson(text: str):
```

Ruff (0.15.11)

[warning] Missing return type annotation for private function `_parse_ndjson`

(ANN202)